src/backend/src/main/java/com/linkedpipes/lpa/backend/ExecutionController.java

```diff
@@ -0,0 +1,45 @@
+package com.linkedpipes.lpa.backend;
+
+import com.linkedpipes.lpa.backend.services.HttpUrlConnector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+@RestController
+public class ExecutionController {
+
+    private static final Logger logger =
+            LoggerFactory.getLogger(DiscoveryController.class);
+
+    private HttpUrlConnector httpUrlConnector = new HttpUrlConnector();
+
+    @RequestMapping("/execution/status")
+    public ResponseEntity<String> getStatus(@RequestParam( value="executionIri") String executionIri) throws IOException{
+        if(executionIri == null || executionIri.isEmpty()) {
+            return new ResponseEntity("Execution IRI not provided.", HttpStatus.BAD_REQUEST);
+        }
+
+        String response = httpUrlConnector.sendGetRequest(executionIri + "/overview",
+                null, "application/json");
```

Comment:
Readability tip, if possible, explicitly name parameters like "null", "true", "1". Those are called magic constants and without going into impl./docs, you have no idea, what that parameters should have been.

Reply:
To clarify, e.g. in C#, I can do

`httpUrlConnector.sendGetRequest(executionIri + "/overview", data: null, mimetype: "application/json")`

if data and mimetype are the correct parameter names